main_app/urls.py

```diff
@@ -8,5 +10,9 @@
     path('detail/<int:game_id>', DetailPageView.as_view(), name='detail_page'),
     path('registration', RegisterPageView.as_view(), name='register_page'),
     path('logout', LogoutPageView.as_view(), name='logout'),
-    path('login', LoginPageView.as_view(), name='login_page')
+    path('login', LoginPageView.as_view(), name='login_page'),
+    path('user', UserPageView.as_view(), name='user_page'),
+    re_path('^activate/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})$',
```

Comment:
Hey, let's rewrite it to custom path converters:
https://docs.djangoproject.com/en/2.2/topics/http/urls/#registering-custom-path-converters
You can place them somewhere in `utils.py`